packages/Calendar/src/polyfills.js

```diff
@@ -0,0 +1,3 @@
+if (!window.crypto) {
```

Comment:
There is a package to manage this kind of things, https://github.com/acl-services/paprika/tree/master/packages/helpers/src/dom

There are other components using the nanoid generator as well please find those in the repo and include the small polyfill as well. 

🙏